pages/index.tsx

```diff
@@ -66,94 +83,104 @@ function	Index(): ReactNode {
 					<div className={'grid grid-cols-2 flex-row gap-4 space-x-0 md:flex md:gap-0 md:space-x-4'}>
-						<label
-							htmlFor={'checkbox-endorsed'}
-							className={'flex w-fit cursor-pointer flex-row items-center rounded-lg bg-neutral-200/60 p-2 font-mono text-sm text-neutral-500 transition-colors hover:bg-neutral-200'}>
-							<p className={'pr-4'}>{'Show endorsed only'}</p>
-							<input
-								type={'checkbox'}
-								id={'checkbox-endorsed'}
-								className={'ml-2 rounded-lg'}
-								checked={settings.shouldShowOnlyEndorsed}
-								onChange={(): void => set_settings({...settings, shouldShowOnlyEndorsed: !settings.shouldShowOnlyEndorsed})} />
-						</label>
-
-						<label
-							htmlFor={'checkbox-anomalies'}
-							className={'flex w-fit cursor-pointer flex-row items-center rounded-lg bg-neutral-200/60 p-2 font-mono text-sm text-neutral-500 transition-colors hover:bg-neutral-200'}>
-							<p className={'pr-4'}>{'Show anomalies only'}</p>
-							<input
-								type={'checkbox'}
-								id={'checkbox-anomalies'}
-								className={'ml-2 rounded-lg'}
-								checked={settings.shouldShowOnlyAnomalies}
-								onChange={(): void => set_settings({...settings, shouldShowOnlyAnomalies: !settings.shouldShowOnlyAnomalies})} />
-						</label>
-
-						<label
-							htmlFor={'checkbox-translations'}
-							className={'flex w-fit cursor-pointer flex-row items-center rounded-lg bg-neutral-200/60 p-2 font-mono text-sm text-neutral-500 transition-colors hover:bg-neutral-200'}>
-							<p className={'pr-4'}>{'Show missing translations'}</p>
-							<input
-								type={'checkbox'}
-								id={'checkbox-translations'}
-								className={'ml-2 rounded-lg'}
-								checked={settings.shouldShowMissingTranslations}
-								onChange={(): void => set_settings({...settings, shouldShowMissingTranslations: !settings.shouldShowMissingTranslations})} />
-						</label>
-
-						<span
-							className={'col-span-2 flex w-full flex-row items-center overflow-hidden rounded-lg bg-neutral-200/60 font-mono text-sm text-neutral-500 transition-colors md:w-fit'}>
-							<p className={'pr-4 pl-2'}>{'Version'}</p>
-
-							<div className={'divide-primary flex flex-row'}>
-								<label className={'cursor-pointer p-2 transition-colors hover:bg-neutral-200'} htmlFor={'checkbox-vaults-all'}>
-									{'All'}
+						<Dropdown
+							defaultOption={OPTIONS[0]}
+							options={OPTIONS}
+							selected={selectedOption}
+							onSelect={(option: TOption): void => {
+								set_selectedOption(option);
+								set_settings({...settings, shouldShowEntity: option.value});
+							}} />
+						{settings.shouldShowEntity === 'vaults' && (
+							<>
+								<label
+									htmlFor={'checkbox-endorsed'}
+									className={'flex w-fit cursor-pointer flex-row items-center rounded-lg bg-neutral-200/60 p-2 font-mono text-sm text-neutral-500 transition-colors hover:bg-neutral-200'}>
+									<p className={'pr-4'}>{'Endorsed only'}</p>
```

Comment:
Removed the word "Show" because otherwise these filters will take 2 lines, happy to discuss a different UI approach for the filters as we might have more of them in the future.